docs/configuration.rst

```diff
@@ -193,6 +225,54 @@ HTTP Adapter
   .. _urllib3.Retry: https://urllib3.readthedocs.io/en/stable/reference/urllib3.util.html
 
 
+S3 Adapter
+**********
+
+This adapter can be used only to download files from `S3 Cloud Storage Service`_. It is intended to be used to configure
+S3 buckets as mirror for track file downloads.
+
+It requires `Boto3 Client`_ to be installed and it accepts only URLs with the following format::
```

Comment:
I think this is too generic. We have specific instructions for `boto3` installation: https://github.com/elastic/rally/blob/5b0ca49fa293014424cfc7200db322a15f2cf7d5/docs/install.rst?plain=1#L101-L104

which makes use of: https://github.com/elastic/rally/blob/5b0ca49fa293014424cfc7200db322a15f2cf7d5/pyproject.toml#L97-L105

We want specific `boto3` version, not just anything which can be found in generic AWS documentation.